pages/mapping.js

```diff
@@ -101,32 +102,43 @@ export default function mapping() {
 
 	function handleNextSelected() {
 		var countryCounts = gameStats.country_counts;
+		const mappedCount = gameStats.mapped_count;
 		if (countryCounts[question.country_code]) {
 			countryCounts[question.country_code] = countryCounts[question.country_code] + 1;
 		} else {
 			countryCounts[question.country_code] = 1;
 		}
-		setGameStats({ country_counts: countryCounts });
+		setGameStats({ country_counts: countryCounts, mapped_count: mappedCount + 1 });
 
 		if (counter < questions.length) {
 			setCounter(counter + 1);
 			setLocationResults({ yes_count: 0, no_count: 0, maybe_count: 0, total_count: 0});
+	  	} else {
+	  		fetchUserStats();
```

Comment:
Based on your PR comments, I believe you didn't intend to keep these 3 lines here

Reply:
Yes. Thanks.